run.py

```diff
@@ -0,0 +1,147 @@
+import sys
+import os
+from datetime import datetime, timedelta
+
+from adsputils import setup_logging, load_config
+
+import argparse
+
+from config import PLANETARYNAMES_PIPELINE_ACTION
+from adsplanetnamepipe import tasks
+from adsplanetnamepipe.utils.common import EntityArgs
+
+proj_home = os.path.realpath(os.path.dirname(__file__))
+config = load_config(proj_home=proj_home)
+
+app = tasks.app
+logger = setup_logging('run.py')
+
+def map_input_param_to_action_type(input_param):
+    """
+
+    :param input_param:
+    :return:
+    """
+    try:
+        action = PLANETARYNAMES_PIPELINE_ACTION[input_param]
+        return action
+    except (KeyError, ValueError):
+        logger.error(f"Invalid action type: {input_param}")
+        return None
+
+def verify_args(args):
+    """
+
+    :param args:
+    :return:
+    """
+    if args.target and args.feature_type:
+        return args.target, args.feature_type, app.get_feature_name_entities(args.target, args.feature_type)
+    if args.target and args.feature_name:
+        return args.target, app.get_feature_type_entity(args.target, args.feature_name), [args.feature_name]
+    return args.target, args.feature_type, [args.feature_name]
+
+def get_date(days):
+    """
+
+    :param days:
+    :return:
+    """
+    if not days:
+        return None
+
+    target_date = datetime.now().date() - timedelta(days=days)
+    target_datetime = datetime.combine(target_date, datetime.now().time())
+    return target_datetime
+
+# python run.py -t Mars -f "Albedo Feature" -a collect
+# python run.py -t Mars -n Amenthes -a collect
+# python run.py -t Mars -n Amenthes -a remove_all_but_last
+# python run.py -t Mars -n Amenthes -a remove_the_most_recent
+# python run.py -t Mars -n Amenthes -a add_keyword_to_knowledge_graph -k basin -t Moon -f Apollo
+# python run.py -t Mars -n Amenthes -a remove_keyword_from_knowledge_graph
+# python run.py -t Mars -n Amenthes -a retrieve_identified_entities -c 0.75
+# TODO: add a command to run all the feature types for all celestial bodies for collect step
+# TODO: add a command to run all the feature names since d days ago
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='Identify Planetary Nomenclature')
+    parser.add_argument('-a', '--action', help='one of the followings: collect, identify, end_to_end (collect followed by identify).')
+    parser.add_argument('-t', '--target', help='target (ie, Moon, Mars, etc) is required. Has to be capitalized.')
+    parser.add_argument('-f', '--feature_type', help='feature type (ie, Crater or Albedo Feature), is one of the options, either feature type or feature name are required, but no both. Has to be capitalized and in singular form.')
+    parser.add_argument('-n', '--feature_name', help='feature name (ie Apollo, Atlas), is one of the options, either feature type or feature name are required, but no both. Has to be capitalized.')
```

Comment:
yes, in database, and init in alembic.

Reply:
regarding second bullet point, yes, I would like to deploy to dev and you guys tested it. this needs lots of resources. I did some ablation testing to see what I can eliminate, and I was not to eliminate any of the libraries.
I like to populate the database, collect data for knowledge graph, this is a one time thing per feature name, then it can be tested for identifying unknown entities, which is a regular operation.

Reply:
OK, so moving the password for the Brain to local_config.
That should be it for this so closing it as done.